coralproject/talk
# Flag username

## What does this PR do?

Allows a user to flag a username or comment and provide details.

## How do I test this PR?

1. Click "Report" underneath a comment.
2. Select whether to report a comment or username. Click continue.
3. Select a reason or enter one via "other." Click continue.
4. If you flagged the comment, the flag should be red.
5. If you flagged the username, it should stay black.
6. Check the network panel, action should be posted to the server at `/v1/api/comments/:comment_id/actions` or `/v1/api/users/:user_id/actions`
